.gitignore

```diff
@@ -1,5 +1,3 @@
 *.DS_Store
-/build/output
```

Comment:
Built output should not be included into repository. When installing using npm it probably should build it as part of installation.